custom_components/smartthinq_sensors/translations/hr.json

```diff
@@ -0,0 +1,40 @@
+{
+    "config": {
+        "abort": {
+            "single_instance_allowed": "Samo jedna konfiguracija SmartThinQ LGE Senzora je dozvoljena.",
+            "no_smartthinq_devices": "Nije pronađen niti jedan SmartThinQ uređaj. Podešavanje komponente prekinuto."
+        },
+        "error": {
+            "invalid_region": "Neispravan format regije.",
+            "invalid_language": "Neispravan format jezika.",
+            "invalid_url": "Neispravan URL za preusmjeravanje.",
+            "invalid_credentials": "Neispravne SmartThinQ vjerodajnice."
+        },
+        "step": {
+            "user": {
+                "data": {
+                    "region": "Kod za državu",
+                    "language": "Kod za jezik"
+                },
+                "description": "Odaberite podatke o lokalizaciji računa SmartThinQ.",
+                "title": "SmartThinQ LGE Senzori"
+            },
+            "url": {
+                "data": {
+                    "login_url": "SmartThinQ URL za prijavu",
+                    "callback_url": "URL za preusmjeravanje"
+                },
+                "description": "Upotrijebite URL u prvom polju za prijavu na SmartThinQ sa svojim vjerodajnicama, a u drugo polje zalijepite URL na koji je preglednik preusmjeren poslije prijave.",
```

Comment:
Can you add refence to the survey as done in the english translation:

```
"description": "Use the URL in the first field to perform login to SmartThinQ with your credentials, then paste the URL where the browser is redirected after the login in the second field. Share feedback about your setup [here](https://git.io/JU166).",
```

Reply:
Done!